File: LeetCode/java/1332_Remove_Palindromic_Subsequences.java
class Solution {
    boolean isPalindrome(String s) {
        StringBuilder rev = new StringBuilder(s);
        rev.reverse();

        if (s.equals(rev.toString())) {
            return true;
        }

        return false;
    }

    public int removePalindromeSub(String s) {
        if (isPalindrome(s)) {
            return 1;
        }

        return 2;
    }
}

File: LeetCode/cpp/1332_Remove_Palindromic_Subsequences.cpp
class Solution {
public:
	bool is_palindrome(string s) {
		string rev_s(s);
		reverse(rev_s.begin(), rev_s.end());

		if (s == rev_s) {
			return true;
		}

		return false;
	}

	int removePalindromeSub(string s) {
		if (is_palindrome(s)) {
			return 1;
		}

		return 2;
	}
};